schema-engine/connectors/schema-connector/src/error.rs

```diff
@@ -75,6 +75,12 @@ impl ConnectorError {
         self.0.user_facing_error.as_ref().map(|err| err.error_code.as_ref())
     }
 
+    /// The error message. Unlike the [`Display`] implementation, this does not include the source
+    /// and context.
+    pub fn message(&self) -> Option<&str> {
```

Comment:
Used in `db_push_on_cockroach_db_with_postgres_provider_fails`, `cockroach_introspection_with_postgres_provider_fails` and `multi_schema_not_implemented_on_mysql` tests to make it possible to use snapshot tests there (otherwise snapshots would be unstable because of file names and line numbers).